cloudify_agent/installer/script.py

```diff
@@ -226,13 +168,12 @@ def install_script_download_link(cloudify_agent=None, **_):
 
 
 def init_script(cloudify_agent=None, **_):
+    # back-compat
     script_builder = _get_script_builder(cloudify_agent=cloudify_agent)
-    return script_builder.init_script()
+    return script_builder.install_script()
 
 
-def init_script_download_link(cloudify_agent=None, **_):
-    script_builder = _get_script_builder(cloudify_agent=cloudify_agent)
-    return script_builder.init_script_download_link()
+install_script = init_script
```

Comment:
Doesn't this entirely shadow install_script?

Reply:
we used to have both `init_script` and `install_script` here, which emitted the linux.sh/linux-download.sh - both eventually did the same thing.

I keep both functions on this level for some "back-compat" - we don't need to, but eh, why not. This is mostly only called from `ctx.plugin.init_script()` (which calls `.init_script` from here)

Anyway, shadow - not really, there's no other function named `install_script` here

Reply:
But you defined install_script earlier in the commit (or left it defined) but then just overrode it here- unless I misread?

Reply:
nah, there's the `install_script` method on the `AgentInstallationScriptBuilder`, but this is a module-level function (which eventually calls that method)